src/web/lib/getCommentContext.ts

```diff
@@ -101,9 +91,15 @@ export const getCommentContext = async (
 ): Promise<CommentContextType> => {
 	const url = joinUrl([ajaxUrl, 'comment', commentId.toString(), 'context']);
 	const filters = initFiltersFromLocalStorage();
-	const params = buildParams(filters);
+	const paramsObj = buildParams(filters);
+
+	const params = Object.keys(paramsObj)
+		// @ts-ignore
+		// eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+		.map((key) => `${key}=${paramsObj[key]}`)
```

Comment:
`key` will always be a key inferred from `paramsObj`, TS is not able to pick up this logic however so we have to ignore